Android/android-unity-wrapper/build.gradle.kts

```diff
@@ -1,13 +1,16 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
-  repositories {
+  val kotlin_version by extra("1.3.72")
+    repositories {
     google()
     mavenCentral()
     jcenter()
   }
   dependencies {
     classpath("com.android.tools.build:gradle:4.1.2")
     classpath("com.vanniktech:gradle-maven-publish-plugin:0.13.0")
+    classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.20")
```

Comment:
Conflicting Kotlin version in classpath and dependency

Reply:
fixed